packages/host/app/services/store.ts

```diff
@@ -221,11 +249,15 @@ export default class StoreService extends Service {
   }
 
   // This method is used for specific scenarios where you just want an instance
-  // that is _not_ part of the identity map and detached from the store. This
-  // may include things like tests, freestyle usage guide, etc.
-  async getInstanceDetachedFromStore(
-    url: string,
-  ): Promise<CardDef | CardError> {
+  // that is not auto saving and not receiving live updates. This may include
+  // things like tests, freestyle usage guide, instances that we immediately
+  // consume, etc.
+  async peek(url: string): Promise<CardDef | CardError> {
+    let cached = this.identityContext.get(url);
+    if (cached) {
+      this.markForGarbageCollection(url);
```

Comment:
Don’t mark for garbage collection if a resource is associated with this instance